src/main/java/org/jabref/model/entry/field/StandardField.java

```diff
@@ -56,6 +56,7 @@ public enum StandardField implements Field {
     EVENTTITLE("eventtitle"),
     EVENTTITLEADDON("eventtitleaddon"),
     FILE("file", FieldProperty.FILE_EDITOR, FieldProperty.VERBATIM),
+    FJOURNAL("fjournal"),
```

Comment:
Sure this should end up in the StandardField-enum? It's more like an internal cache for the abbreviator

Reply:
I would add it to the JabRef Internal Fields

Reply:
According to https://tex.stackexchange.com/q/272164/9075, this field stems from MathSciNet (https://mathscinet.ams.org/mref) and thus is similar to `eventtitle`, which is also not used that commonly, but available. The field is not JabRef-only. The alternative is to introduce "ExoticField.java" (or similar), which I find redicoulus ^^.

Reply:
Maybe we should distinguish in general between standard fields defined by bibtex/biblatex and other fields or we could treat them in general as so called UnknownFields?

Reply:
Current type hierarchy is as follows:

![grafik](https://user-images.githubusercontent.com/1366654/199559012-370b5501-603f-4de2-ab8b-59299af4dfa9.png)

Based on that, I would create a class `AMSField`. (In consistency to [AMSMath](https://www.ctan.org/pkg/amsmath/))

---

![grafik](https://user-images.githubusercontent.com/1366654/199559414-b0c2862a-e1bb-4471-8fcb-b96fba16040a.png)

Input for https://mathscinet.ams.org/mathscinet-mref

```
Dal Maso, Gianni; Fonseca, Irene; Leoni, Giovanni Nonlocal character of the reduced theory of thin films with higher order perturbations. Adv. Calc. Var. 3 (2010), no. 3, 287-319. 
```

Output

```bibtex
@article {MR2660690,
    AUTHOR = {Dal Maso, Gianni and Fonseca, Irene and Leoni, Giovanni},
     TITLE = {Nonlocal character of the reduced theory of thin films with
              higher order perturbations},
   JOURNAL = {Adv. Calc. Var.},
  FJOURNAL = {Advances in Calculus of Variations},
    VOLUME = {3},
      YEAR = {2010},
    NUMBER = {3},
     PAGES = {287--319},
      ISSN = {1864-8258},
   MRCLASS = {49J45 (35Q74 74G65 74K35)},
  MRNUMBER = {2660690},
MRREVIEWER = {Giovanni Alberti},
       DOI = {10.1515/ACV.2010.012},
       URL = {https://doi.org/10.1515/ACV.2010.012},
}
```